pkg/controller/useraccount/useraccount_controller.go

```diff
@@ -212,38 +221,25 @@ func newUser(userAcc *toolchainv1alpha1.UserAccount) *userv1.User {
 		ObjectMeta: metav1.ObjectMeta{
 			Name: userAcc.Name,
 		},
+		Identities: []string{getIdentityName(userAcc)},
```

Comment:
@alexeykazakov FYI, you setting `Identities` in `newUser()` is enough to established mapping from user to identity and the logic here [link](https://github.com/codeready-toolchain/member-operator/blob/6be0efb12395e5d45ac450e8fbc0eaa55e012570/pkg/controller/useraccount/useraccount_controller.go#L136-L147) is never get executed.

Reply:
It does get executed. This logic is for updating the existing User in case it's missing the mapping.
Example:
1. Create UserAccount and let the controller create User, Identity with mapping.
2. Update the created User and remove the reference to the Identity.
3. The controller will restore the mapping.

You can remove this code and see that the test for updating the user is now failing.